pkg/tools/etcd_helper_test.go

```diff
@@ -724,7 +705,7 @@ func TestGetEtcdVersion_ErrorStatus(t *testing.T) {
 	defer testServer.Close()
 
 	var err error
-	_, _, err = GetEtcdVersion(testServer.URL)
+	_, err = GetEtcdVersion(testServer.URL)
```

Comment:
This is existing, but it is weird to do this rather than `_, err := GetEtcdVersion()` here.